packages/adapter-statsig/package.json

```diff
@@ -66,7 +64,10 @@
     "vite": "5.1.1",
     "vitest": "1.4.0"
   },
-  "peerDependencies": {},
+  "peerDependencies": {
+    "@vercel/edge-config": "~1.2.0",
+    "statsig-node-vercel": "~0.4.0"
+  },
```

Comment:
You can further add this to mark the dependencies as optional

```
  "peerDependenciesMeta": {
    "@vercel/edge-config": {
      "optional": true
    },
    "statsig-node-vercel": {
      "optional": true
    }
  },
```

This way people not using Edge Config won't get warnings